src/Symfony/Component/Form/Tests/Extension/Core/Type/ChoiceTypeTest.php

```diff
@@ -94,6 +100,54 @@ public function testExpandedChoicesOptionsTurnIntoChildren()
         $this->assertCount(count($this->choices), $form, 'Each choice should become a new field');
     }
 
+    public function testChoiceListWithScalarValues()
+    {
+        $view = $this->factory->create('Symfony\Component\Form\Extension\Core\Type\ChoiceType', null, array(
+            'choices' => $this->scalarChoices,
+        ))->createView();
```

Comment:
If you're merging this into 3.0, you can use `ChoiceType::class`, not sure if that's preferable though.